.github/workflows/ci.yml

```diff
@@ -0,0 +1,110 @@
+name: ci
+on: [push, pull_request]
+jobs:
+  linux:
+    name: ubuntu
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Init Git Submodule 
+      run: |  
+        git submodule update --init --recursive
```

Comment:
```yaml
    - name: Checkout repository
      uses: actions/checkout@v2
      with:
        submodules: 'recursive'
```

Reply:
@ilkerhalil I see you've got a second PR that addresses this.  Are either ready for review?

Reply:
@Rjvs Not ready now. I want to check again. I will invite you to be ready.